src/elements/content-sharing/SharingNotification.js

```diff
@@ -6,7 +6,7 @@ import API from '../../api';
 import Notification from '../../components/notification/Notification';
 import NotificationsWrapper from '../../components/notification/NotificationsWrapper';
 import { convertItemResponse } from '../../features/unified-share-modal/utils/convertData';
-import { ACCESS_COLLAB, STATUS_ERROR, TYPE_FILE, TYPE_FOLDER } from '../../constants';
+import { ACCESS_COLLAB, ACCESS_NONE, STATUS_ERROR, TYPE_FILE, TYPE_FOLDER } from '../../constants';
```

Comment:
i am not sure what ACCESS_NONE means. does it actually correspond to a value returned by the API, or is it more of a placeholder when removing?

Reply:
Yup, it's a placeholder. `ShareAccessSelect` also uses `ACCESS_NONE`, so I retained that part of the Item `share()` code to ensure backwards compatibility.